packages/common/driver-definitions/src/driverError.ts

```diff
@@ -90,11 +90,17 @@ export enum DriverErrorType {
      * "fluidInvalidSchema"
      */
     fluidInvalidSchema = "fluidInvalidSchema",
+
     /**
      * Error indicating an API is being used improperly resulting in an invalid operation.
      * ! Should match the value of ContainerErrorType.usageError
     */
     usageError = "usageError",
+
+    /**
+     * File is locked, status code 423
```

Comment:
I'd remove status code reference, as it's SPO specific.
Instead, I'd put more in-depth description of when it happens - storage supports file locking (not through Fluid protocol), and Fluid Framework can't open file (even for reading) when file is locked.
That said, any chance you can follow up with SPO folks to check when this error is thrown, and is it truly the case that file can't be opened even for reading? I missed that originally, but possibly the better design here (if we can open file for reading) is not to throw error to user, but rather return fluid container in read-only state, similar to how we deal with it in other places where file is read-only